ui/pages/invite/[token].tsx

```diff
@@ -0,0 +1,80 @@
+import { useEffect } from "react";
+import { useRouter } from "next/router";
+import { useMutation, gql } from "urql";
+import HappySpinner from "components/HappySpinner";
+
+const JOIN_ROUND = gql`
+  mutation JoinRoundInvitationLink($token: String!) {
+    joinRoundInvitationLink(token: $token) {
+      id
+      round {
```

Comment:
Maybe should add the round id here, graphql tends to yell at me when I don't fetch that.